src/main/java/com/cicdlectures/menuserver/controller/MenuController.java

```diff
@@ -37,4 +42,10 @@
   public MenuDto createMenu(@RequestBody MenuDto menu) {
     return createMenuService.createMenu(menu);
   }
+
+  @DeleteMapping(path = "/menus/{id}")
+  int deleteMenu(@PathVariable Long id) {
```

Comment:
Il faudrait lui spécifier la visibilité.

Reply:
will work on it